data/lessons.json.unvalidated.in

```diff
@@ -264,10 +264,10 @@
                 "input": "console",
                 "mapper": ["shell", "raw_output", {
                     "type": "regex",
-                    "value": "^always-show-log-out$"
+                    "value": "^can-change-accels$"
```

Comment:
We get the keys for another schema, org.gnome.desktop.interface, and this is the key on top here.

The lesson about get/set in the gsettings lesson is about changing the clock-format now. That's https://github.com/endlessm/showmehow/pull/49